internal/pkg/security/security_test.go

```diff
@@ -47,8 +50,6 @@ func TestGetParam(t *testing.T) {
 func TestConfigure(t *testing.T) {
 	test.EnsurePrivilege(t)
```

Comment:
there's a t.Skip around here on master, which should be removed for this to work.

Reply:
it was removed